### ISSUE TITLE
Interface Historical graph

### ISSUE THREAD

**Issue by HoseinKashefi** [ntop/ntopng#2003]
after update to following versions, ntopng collect data and shows it in dashboard section, but in interface historical  doesn't show any traffic graph( it shows 'Active Flows' graph)
ntopng Enterprise Edition v.3.7.180910
Cento v.1.5.180910

![traffic graph](https://user-images.githubusercontent.com/43135531/45291795-6560ab00-b508-11e8-84dd-5b0dea49cafa.jpg)

**Comment by emanuele-f** [comment]
Hi, can you post a screenshot of the timeseries preferences page? Can you see any errors in the ntopng log? Please check out `journalctl -u ntopng`

**Comment by HoseinKashefi** [comment]
as i monitored i didnt find any errors.
here you can see the last startup log:
Starting ntopng high-speed web-based traffic monitoring and analysis tool...
Sep 10 15:03:44 Started ntopng high-speed web-based traffic monitoring and analysis tool.
10/Sep/2018 15:03:44 [Ntop.cpp:1547] Setting local networks to xxxxxxxxxxxxxxxxxxxxxxxxx
10/Sep/2018 15:03:44 [Redis.cpp:127] Successfully connected to redis 127.0.0.1:6379@0
10/Sep/2018 15:03:44 [Redis.cpp:127] Successfully connected to redis 127.0.0.1:6379@0
10/Sep/2018 15:03:44 [NtopPro.cpp:312] [LICENSE] Reading license from /etc/ntopng.license
10/Sep/2018 15:03:44 [NtopPro.cpp:208] [LICENSE] /etc/ntopng.license: found valid Enterprise license
[HostPools.cpp:359] WARNING: JSON Parse error [unexpected end of data] key: ntopng.serialized_host_pools.ifid_0: { "sent": { "packets": 3405351020, "bytes": 4139965681617 }, "rcvd": { "p
10/Sep/2018 15:03:46 [HostPools.cpp:359] WARNING: JSON Parse error [unexpected end of data] key: ntopng.serialized_host_pools.ifid_0: { "sent": { "packets": 3405351020, "bytes": 41399656
10/Sep/2018 15:03:46 [Ntop.cpp:1641] Registered interface tcp://*:5556c [id: 0]
10/Sep/2018 15:03:46 [main.cpp:298] PID stored in file /var/run/ntopng.pid
10/Sep/2018 15:03:47 [Utils.cpp:507] User changed to ntopng
10/Sep/2018 15:03:47 [HTTPserver.cpp:1114] Web server dirs [/usr/share/ntopng/httpdocs][/usr/share/ntopng/scripts]
10/Sep/2018 15:03:47 [HTTPserver.cpp:1117] HTTP server listening on 3000
10/Sep/2018 15:03:47 [main.cpp:383] Working directory: /var/lib/ntopng
10/Sep/2018 15:03:47 [main.cpp:385] Scripts/HTML pages directory: /usr/share/ntopng
10/Sep/2018 15:03:47 [Ntop.cpp:390] Welcome to ntopng x86_64 v.3.7.180910 - (C) 1998-18 ntop.org
10/Sep/2018 15:03:47 [Ntop.cpp:400] Built on CentOS Linux release 7.5.1804 (Core)
10/Sep/2018 15:03:47 [NtopPro.cpp:631] [LICENSE] System Id:        xxxxxxxxxxxxxx
10/Sep/2018 15:03:47 [NtopPro.cpp:632] [LICENSE] Edition:        Enterprise
10/Sep/2018 15:03:47 [NtopPro.cpp:633] [LICENSE] License Type:        Permanent License
10/Sep/2018 15:03:47 [NtopPro.cpp:636] [LICENSE] License:        xxxxxxxxxxxxxxxxxxxxxxxxxxxxx
10/Sep/2018 15:03:47 [NtopPro.cpp:646] [LICENSE] Maintenance:        Until Tue Jul 30 15:13:26 2019 [323 days left]
10/Sep/2018 15:03:47 [PeriodicActivities.cpp:59] Started periodic activities loop...
10/Sep/2018 15:03:47 [PeriodicActivities.cpp:100] Each periodic activity script will use 2 threads
10/Sep/2018 15:03:47 [NetworkInterface.cpp:2564] Started packet polling on interface tcp://*:5556c [id: 0]...
10/Sep/2018 15:03:48 [CollectorInterface.cpp:122] Collecting flows on tcp://*:5556c
[ParserInterface.cpp:1014] WARNING: Invalid message received: your nProbe sender is outdated, data encrypted or invalid JSON?
10/Sep/2018 15:04:53 [ParserInterface.cpp:1014] WARNING: Invalid message received: your nProbe sender is outdated, data encrypted or invalid JSON?
Sep 10 15:04:53 [342B blob data]
Sep 10 15:04:53 [321B blob data]
![timeseries 1](https://user-images.githubusercontent.com/43135531/45294804-c5a81a80-b511-11e8-8c5a-7b76ba1e7cc8.JPG)
![timeseries 2](https://user-images.githubusercontent.com/43135531/45294807-cb9dfb80-b511-11e8-820a-a57032961ba8.JPG)

**Comment by emanuele-f** [comment]
Have you upgrade nprobe to the latest version? Check out the "invalid message received" warning. If you go into the interface settings (cog icon in your first screenshot), is the timeseries creation option checked?

**Comment by HoseinKashefi** [comment]
yes timeseries creation was checked. and as i mentioned at the beginning both cento and ntopng were upgraded to latest version and build.
ntopng Enterprise Edition v.3.7.**180910**
Cento v.1.5.**180910**

**Comment by emanuele-f** [comment]
Do you have Dynamic Interface disaggregation enabled? Can you see the charts for other interfaces? Can you provide access to the ntopng gui or the configuration dump (`Backup Configuration` in the menu) via email at faranda@ntop.org ?

**Comment by HoseinKashefi** [comment]
i provided you an access to ntop gui.you can check it now.

**Comment by emanuele-f** [comment]
Thank you. Can you please edit `/usr/share/ntopng/scripts/lua/modules/timeseries/drivers/rrd.lua` by inserting the following lines above the line `ntop.rrd_update(rrdfile, table.unpack(params))` :

```
  tprint(rrdfile)
  tprint(params)
```

Then, after one minute, you should see something similar to this in the ntopng log:

```
string /var/lib/ntopng/0/rrd/drops.rrd
 table
1 string 1536656224
2 string 0
 string /var/lib/ntopng/0/rrd/bytes.rrd
 table
1 string 1536656224
2 string 28529830
```

Please post a portion of the output. Can you see `bytes.rrd` in your output?

**Comment by HoseinKashefi** [comment]
[rrdlog.txt](https://github.com/ntop/ntopng/files/2370435/rrdlog.txt)
Please find the attachment for output.

**Comment by emanuele-f** [comment]
This should be fixed in 433cbb3142586c50f989f32c03f2c997329f8056 . The new package will be available in one hour.

**Comment by HoseinKashefi** [comment]
packages have been updated and now ntopng works fine.
thanks a million.
